opm/autodiff/BlackoilModelBase.hpp

```diff
@@ -323,7 +323,7 @@ namespace Opm {
         bool use_threshold_pressure_;
         V threshold_pressures_by_connection_;
 
-        SimulatorData sd_;
+        mutable SimulatorData sd_;
```

Comment:
rsSat/rvSat is set/updated within variableStateExtractVars(...) const, which this is made mutable